site/gatsby-site/src/utils/checklists.js

```diff
@@ -0,0 +1,121 @@
+import { classy } from 'utils/classy';
+import { faShield, faWarning } from '@fortawesome/free-solid-svg-icons';
+
+const abbreviatedTag = (tag) => tag.replace(/^.*:/g, '');
+
+const risksEqual = (risk1, risk2) => {
+  if (risk1.generated != risk2.generated) {
+    return false;
+  }
+  if (risk1.generated && risk2.generated) {
+    return risk1.title == risk2.title;
+  }
+  if (!risk1.generated && !risk1.generated) {
+    return risk1.id == risk2.id;
+  }
+};
+
+const Label = classy('label', 'mb-1 block');
+
+const emptyRisk = () => ({
+  id: Math.random().toString(36).slice(-10),
+  title: 'Untitled Risk',
+  tags: [],
+  risk_status: 'Not Mitigated',
+  risk_notes: '',
+  severity: '',
+  likelihood: '',
+  precedents: [],
+  touched: false,
+  generated: true,
+  startClosed: false,
+});
+
+const removeTypename = (obj) => {
```

Comment:
This function and others deserve unit tests for input/output.

If you want to follow up in a separate PR, please add issues and link it in a TODO in the code here.

Reply:
https://github.com/responsible-ai-collaborative/aiid/issues/2301